tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -214,18 +215,21 @@ await Task.Run(() =>
                 if (!wrapperResult.Succeeded)
                 {
                     _log.Error(wrapperResult.ExtendedError, "Failed to register folder for source control integration");
-                    return;
+                    return new SourceControlValidationResult(ResultType.Failure, ErrorType.RegistrationWithFileExplorerFailed, wrapperResult.ExtendedError, _stringResource.GetLocalized("RegistrationErrorWithFileExplorer"), _stringResource.GetLocalized("RegistrationErrorWithFileExplorer"));

```

Comment:
DiagnosticText shouldn't be explicitly localized. If we don't have anything more internally useful to say in DiagnosticText, I think it's fine to leave null.